Source/Shaders/ModelExperimental/MaterialStageFS.glsl

```diff
@@ -0,0 +1,214 @@
+czm_modelMaterial defaultModelMaterial() {
```

Comment:
Update the formatting on this file. 4 space tabs, K&C style braces based on our style guide.